avocado/core/test.py

```diff
@@ -774,6 +784,9 @@ def skip(self, message=None):
         """
         raise exceptions.TestSetupSkip(message)
 
+    def cancel(self, message=None):
```

Comment:
A docstring that will end up being shown in the API docs section is very much welcome here.